### DISCUSSION TITLE
highlight group NvimTreeWinSeparator doesn't work when view.side = "right"

### DISCUSSION THREAD

**Discussion by gmr458** [nvim-tree/nvim-tree.lua#2284] [Q&A]
I use `NvimTreeWinSeparator` to hide the `WinSeparator` and made NvimTree look a little bit better, this works fine with `view.side = "left"`.

![20230618_19h23m36s_grim](https://github.com/nvim-tree/nvim-tree.lua/assets/53343401/0840aa75-7874-4736-a940-cb715068a86d)

When I use `view.side = "right"` this happens:

![20230618_19h25m44s_grim](https://github.com/nvim-tree/nvim-tree.lua/assets/53343401/9ad03ff5-52ef-4dd8-a4a9-6d2d8a077fc4)

Is there a way to hide that line?

**Comment by alex-courtis** [comment]
`NvimTreeWinSeparator` applies to the nvim-tree window only. Opening another window to the right will show it:

`:hi NvimTreeWinSeparator guifg=red guibg=green`

![20230625_111918](https://github.com/nvim-tree/nvim-tree.lua/assets/1505378/558cce03-c0e5-443a-80c5-4bc9df9a92d0)

Try setting `WinSeparator` for (all?) other windows.
